exercises/APE8/0801.tex

```diff
@@ -0,0 +1,160 @@
+With the knowledge of the previous courses, we know that we are tempted to randomize the strategies, that is, choose parameters $\lambda$ and $\mu$, then play with strategy $\parent{\lambda a_1, \parent{1 - \lambda} b_1}$ for player 1 and $\parent{\mu a_2, \parent{1 - \mu} b_2}$ for player 2. However, now we will randomize over the outcomes, hence we will choose parameters $\alpha$, $\beta$, $\gamma$ and $\delta$ for outcomes of $\parent{a_1, a_2}$, $\parent{a_1, b_2}$, $\parent{b_1, a_2}$ and $\parent{b_1, b_2}$, respectively. The optimization problem is
+
+\begin{equation*}
+    \begin{aligned}
+    \underset{}{\text{max }} &
+    f \parent{\alpha, \beta, \gamma, \delta} & & \\
+    \text{such that  } &
+        \alpha, \beta, \gamma, \delta \geq 0 & \\
+        & \alpha + \beta + \gamma + \delta = 1 & \\
+        & \alpha, \beta, \gamma, \delta \in C . &
+    \end{aligned}
+    \end{equation*}
+
+The set $C$ is the set containing all the parameters such that the equilibrium is correlated. If we look at the table, we see that the game is symmetric. We can prove that the equilibrium is invariant under transformation, since the set of feasible solutions is convex. Hence we can say that $\beta = \gamma$. We have thus transformed a four-dimensional problem into a three-dimensional problem. Furthermore, we can delete the parameter $\delta$ because $\alpha + 2 \beta + \delta = 1$, hence $\delta = 1 - \alpha - 2 \beta$. We have thus transformed a three-dimensional problem into a two-dimensional problem.
```

Comment:
"We can prove that the equilibrium is invariant under transformation, since the set of feasible solutions is convex" is a bit confusing. What do you mean ?
I would write "We can prove that there exist an optimal solution with $\beta = \gamma$ equilibrium using the convexity of the problem"